berza/src/main/java/rs/edu/raf/banka/berza/controller/FuturesUgovoriPodaciController.java

```diff
@@ -33,16 +33,4 @@ public ResponseEntity<?> getFuturesUgovor(@PathVariable String symbol){
 //        return ResponseEntity.ok(futuresUgovoriPodaciService.getFuturesUgovor(id));
 //    }
 
-    @GetMapping(value = "/timeseries/{type}/{symbol}", produces = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<?> getAkcijeTimeseries(@PathVariable String type, @PathVariable String symbol){
-        if(type == null || type.isBlank()) {
-            return ResponseEntity.badRequest().build();
-        }
-        if(symbol == null || symbol.isBlank()) {
-            return ResponseEntity.badRequest().build();
-        }
-        return ResponseEntity.ok(futuresUgovoriPodaciService.getFuturesTimeseries(type, symbol));
-    }
```

Comment:
Izbačeno zato što je profesor tražio da se izbaci, ovo je na frontendu obrisano još za prvu kontrolnu tačku.